### PR TITLE
Add the email validation code to a script.js file

### PR DESCRIPTION
**Pull request by Nedramrez**
- Add the email validation code to a script.js file

### CODE REVIEW

File: script.js [hunk 1]
```diff
@@ -118,3 +118,18 @@ const detailsPp = function (detail) {
...
 };
+
+/* The Email Validation */
+document.querySelector('.submit').addEventListener('click', (event) => {
+  const email = document.getElementById('mail').value;
+  const lowcaseError = document.querySelector('.lowcase-error');
+  const isLowerCase = (str) => str === str.toLowerCase();
+  if (!isLowerCase(email)) {
+    event.preventDefault();
+    lowcaseError.innerHTML = 'Please use lowercase letters and proper mail pattern';
```
Comment (omarsalem7) [inline script.js:129]: - [ ] Kindly consider showing an error message to the user near the submit button not between inputs fields.

**Review by omarsalem7 — CHANGES_REQUESTED** [review]
Hi @Nedramrez 😊, 

Good job so far! 👏
You have done well reaching this point of the milestone, but there are some issues that you still need to work on to go to the next project but you are almost there! 

you-are-near

## Highlights
- Descriptive pull request ✔️
- No linter errors ✔️

## Required Changes ♻️
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by AmaduKamara — APPROVED** [review]
# Status: Approved 🟢🟢🟢 

Hi @Nedramrez,

YesYouAreGIF

Your project is complete!  There is nothing else to say other than... it's time to merge it: ship it:
Congratulations! 🎉 

## Highlights

✅ Correct use of Github workflow
✅ No linter error
✅ Good commit messages 👍 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Comment by Nedramrez** [comment]
Thank you very much!

**Pull request merged**